sbt-version-policy/src/main/scala/sbtversionpolicy/SbtVersionPolicyMima.scala

```diff
@@ -74,9 +114,18 @@ object SbtVersionPolicyMima extends AutoPlugin {
     versionPolicyFirstVersion := None,
   )
 
+  override def buildSettings = Def.settings(
+    coursierCredentialsFile := file {
+      if (sys.props("os.name").toLowerCase.contains("mac"))
+        s"${sys.props("user.home")}/Library/Application Support/Coursier/credentials.properties"
+      else
+        s"${sys.props("user.home")}/.config/coursier/credentials.properties"
```

Comment:
are we sure there isnt already this default path defined somewhere in coursier?

Reply:
There is a method for the config directories, `coursier.paths.CoursierPaths.configDirectories()`. But it's shaded by sbt, and it errors at compile time with:
```
[error] java.lang.AssertionError: assertion failed:
[error]   No RuntimeVisibleAnnotations in classfile with ScalaSignature attribute: package object coursier 
```